enterprise/server/backends/gcs_cache/gcs_cache.go

```diff
@@ -60,18 +61,29 @@ func NewGCSCache(bucketName, projectID string, ageInDays int64, opts ...option.C
 	return g, nil
 }
 
+func (g *GCSCache) bucketExists(ctx context.Context, bucketName string) (bool, error) {
+	traceCtx, spn := tracing.StartSpan(ctx)
```

Comment:
nit: can we override the name "ctx" everywhere instead of creating a new name "traceCtx"?

I think in all cases in this CL we want to use the new ctx, so this prevents someone else from mistakenly using the wrong one.

Reply:
I can change it mostly, but the places where we use two spans in one function would be bad to override the context, no? As we would then be using the context from the previous span (that had ended) to make a new span, which seems wrong?

Reply:
I guess maybe more accurately, I think I should only use the new context so long as the related span is active

Reply:
ah yes, that one i agree, you're right!